src/binding/events.js

```diff
@@ -88,23 +90,29 @@ Events.prototype.emit = function (name, arg0, arg1, arg2, arg3, arg4, arg5, arg6
     if (this._suspendEvents) return;
 
     var events = this._events[name];
-    if (! events)
-        return this;
-
-    events = events.slice(0);
-
-    for (var i = 0; i < events.length; i++) {
-        if (! events[i])
-            continue;
-
-        try {
-            events[i].call(this, arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7);
-        } catch (ex) {
-            console.info('%c%s %c(event error)', 'color: #06f', name, 'color: #f00');
-            console.log(ex.stack);
+    if (events) {
+        events = events.slice(0);
+
+        for (var i = 0; i < events.length; i++) {
+            if (! events[i])
+                continue;
+
+            try {
+                events[i].call(this, arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7);
+            } catch (ex) {
+                console.info('%c%s %c(event error)', 'color: #06f', name, 'color: #f00');
+                console.log(ex.stack);
+            }
         }
     }
 
+    if (this._additionalEmitters.length) {
+        const emitters = this._additionalEmitters.slice();
```

Comment:
Should this match line 94 and be `this._additionalEmitters.slice(0);`?

Reply:
don't think it matters